lib/suspenders/generators/app_generator.rb

```diff
@@ -172,6 +174,13 @@ def setup_git
       end
     end
 
+    def git_first_commit
+      if !options[:skip_git]
+        say 'Init commit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.